lib/airbrake/utils/params_cleaner.rb

```diff
@@ -106,11 +106,11 @@ def clean_unserializable_data(data, stack = [])
             data.to_hash.inject({}) do |result, (key, value)|
               result.merge!(key => clean_unserializable_data(value, stack + [data.object_id]))
             end
-          elsif data.respond_to?(:collect!)
+          elsif data.respond_to?(:collect!) && !data.frozen?
```

Comment:
This is fixed with https://github.com/airbrake/airbrake/commit/0939290315752a5de5cfd226ad65583f1d06b1f1